website/static/js/osfHelpers.js

```diff
@@ -857,6 +857,78 @@ function onScrollToBottom(element, callback) {
     });
 }
 
+/**
+ *  The following jquery functions are to fix the bootstrap affix issue
+ *  where having a top offset and a bottom offset can cause conflicts when the page is refreshed.
+ *  In this cause it caused the user settings side navigation bar to stick to the bottom of the page
+ *  when refreshed. The first function checks if the page has been shrunk to the mobile settings or not.
+ *  The second function affixes the element to the top of the screen window and the top of the page footer.
+ *  @param {String} panelID:
+ *  @param {Integer} panel_offset:
+ */
+var stickIt = function stickIt(panelID, panel_offset){
+    this.elementID = panelID;
+    this.offsetElement = panel_offset;
+
+    var lastWidth;
+    $(window).load(function(){
+        if($(window).width() > 769){
```

Comment:
Magic numbers are generally a bad idea. Can you re-work this to use bootstrap? If we're looking for transitions from states, we should be checking what states transitions will cause a problem and maybe looking at what the values for those transitions are in a more generic way. That being said, that sounds like a lot of trouble for not much effort.

If we keep it in osfHelpers, it should probably be pretty generic, even if it's just a matter of passing some parameters from the calling javascript so that different pages with different rules will be able to pass that in properly. Alternately, we could move this function to the page that would use it (if it's only in one place) and maybe define the number as a constant somewhere in the js.

I lean towards the "passing in a parameter" approach, but think about what makes sense to you.